File: 은지/LV1/2016년.py
def solution(a, b):
    Month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    Week = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU']
    # 일 수 더하기
    day = b-1
    # 달의 일 수 더하기
    for i in range(a-1):
        day += Month[i]
    # 변경 요일 탐색
    day %= 7
    answer = Week[day]
    return answer

File: 은지/LV1/2016년.js
function solution(a, b) {
  var answer = '';
  const month = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
  const week = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU']
  var day = b-1
  for (let i=0; i<a-1; i++) {
    day += month[i]
  }
  day %= 7
  answer = week[day]
  return answer;
}